src/pages/dac_dataset_catalog/CustomDatasetCatalog.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import _ from 'lodash';
+import '../DatasetCatalog';
+import SampleLogoIcon from '../../images/sample_dac_logo.png';
+import BroadLogoIcon from '../../images/broad_logo.png';
+import NotFound from '../NotFound';
+import DatasetCatalog from '../DatasetCatalog';
+import './CustomDatasetCatalog.css';
+
+
+/**
+ * Variants can be added and must include the following:
+ * path: the last element of the path that will identify this catalog variant
+ * icon: an icon for the catalog variant.  It should have as close to a 4x1 ratio as possible
+ * dacName: the name of the DAC that this catalog is for
+ * dacFilter: filter method to get the datasets to show on the catalog by default. Return true to include
+ * the referenced dataset.  The callback takes:
+ *     dataset: the datataset to evaluate.  The content of the dataset may vary, as a result to see what
+ *              fields are available, open the developer console and go to the sample page.  The datasets will
+ *              echoed to the console in dev or local mode
+ * colorKey: the key of the color scheme for the catalog.  See the CustomDatasetCatalog.css file for
+ *   instructions on how to use the colorKey
+ */
```

Comment:
Thinking toward the future ...
If we extracted this logic into an API call in consent that returned a simple json response of variants, we could open up an admin interface to create/manage the variants which would give Jonathan the ability to create/manage these on the fly. Lots of +'s/-'s to consider, but I think this is a great foundation to work from.